src/script_lua.c

```diff
@@ -50,6 +50,7 @@ static char *libraries_allow_list[] = {
     "math",
     "table",
     "struct",
+    "os",
```

Comment:
import os library would be dangerous, e.x os.execute, os.rename

Reply:
Yup, it could be replaced with "ros" or something else. The initial intention to name it 'os' is to conform to the concept of daily use.